op-service/testlog/testlog.go

```diff
@@ -159,11 +159,21 @@ func (l *logger) flush() {
 
 	scanner := bufio.NewScanner(l.buf)
 	for scanner.Scan() {
-		l.t.Logf("%*s%s", padding, "", scanner.Text())
+		l.internalFlush("%*s%s", padding, "", scanner.Text())
 	}
 	l.buf.Reset()
 }
 
+func (l *logger) internalFlush(format string, args ...any) {
+	defer func() {
```

Comment:
I think this might need a `l.t.Helper()` call to make the log file-prefix accurate

Reply:
done